packages/hint-no-disallowed-headers/src/_locales/en/messages.json

```diff
@@ -11,6 +11,30 @@
         "description": "Report message when the 'server' header contains more than the server name",
         "message": "The 'server' header should only contain the server name."
     },
+    "disallowedExpiresHeader": {
+        "description": "Report message when the response includes the Expires header",
+        "message": "The 'Expires' header should not be used, 'Cache-Control' should be preferred."
```

Comment:
Thanks again for breaking these out! I really appreciate having actionable messages, especially when we're suggesting developers use an alternative (as opposed to simply removing a header).